presto-main/src/main/java/com/facebook/presto/type/LikeFunctions.java

```diff
@@ -94,6 +96,63 @@ public static Regex likePattern(@SqlType("varchar(x)") Slice pattern, @SqlType("
         return likePattern(pattern.toStringUtf8(), getEscapeChar(escape), true);
     }
 
+    public static boolean isLikePattern(Slice pattern, Slice escape)
+    {
+        String stringPattern = pattern.toStringUtf8();
+        if (escape == null) {
+            return stringPattern.contains("%") || stringPattern.contains("_");
+        }
+
+        String stringEscape = escape.toStringUtf8();
+        checkCondition(stringEscape.length() == 1, INVALID_FUNCTION_ARGUMENT, "Escape must be empty or a single character");
```

Comment:
I think he was confused about the text "Escape must be empty or a single character" which should be "Escape must be a single character"